Tasks/NuGetCommandV2/Tests/L0.ts

```diff
@@ -9,8 +9,8 @@ describe('NuGetCommand Suite', function () {
     after(() => {
     });
 
-    it('restore single solution', (done: MochaDone) => {
-        this.timeout(1000);
+    it('restore single solution', (done: Mocha.Done) => {
+        this.timeout(10000);
```

Comment:
Why did you have to change the timeouts?

Reply:
I was experimenting, forgot to change it back to 1000.  Changed it back to 1000 now.